pinot-common/src/main/java/org/apache/pinot/common/utils/tls/RenewableTlsUtils.java

```diff
@@ -216,6 +218,20 @@ static void enableAutoRenewalFromFileStoreForSSLFactory(SSLFactory sslFactory, S
           throw new RuntimeException(e);
         }
       });
+
+      // Reloading SSL Factory blindly every day in a new scheduled thread
```

Comment:
Add the reason why the additional scheduled thread is added here.

Reply:
Done